.github/workflows/release-trigger.yaml

```diff
@@ -22,5 +24,8 @@ jobs:
         with:
           node-version: '20.x' # semantic-release requires Node version 20.8.1 or higher
       - name: semantic-release
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: |
-          npx semantic-release@18
\ No newline at end of file
+          npm install conventional-changelog-conventionalcommits -D
+          npx semantic-release@24
```

Comment:
The version proved to be important, so maybe we can add a not about why this one?

Reply:
Yea, this one is just the latest major. It was the recommended approach they had. They just had the incorrect version documented.

By linking to major we won't accidentally pick up a breaking change, but otherwise we'll get patch versions etc asap.